docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -1,668 +1,90 @@
+- **If using a SAML-based solution like Okta**, users must be assigned to the Dagster app in the SSO portal to log in. By default, users will be granted Viewer permissions on each deployment. The default role can be adjusted by modifying the [`sso_default_role` deployment setting](/dagster-cloud/managing-deployments/deployment-settings-reference#sso-default-role).
 
-<!-- ![Overriding the Viewer user role for a code location](/images/dagster-cloud/user-token-management/code-location-override.png) -->
+1. Sign in to your Dagster Cloud account.
+2. Click the **user menu (your icon) > Cloud Settings**.
+3. Click the **Users** tab.
+4. Click **Add new user.**
+5. In the **User email** field, enter the user's email address.
+6. Click **Add user**. The user will be added to the list of users.
 
-<Image
-alt="Overriding the Viewer user role for a code location"
-src="/images/dagster-cloud/user-token-management/code-location-override.png"
-width={700}
-height={499}
-/>
+After the user is created, you can [add the user to teams and assign user roles for each deployment](#managing-users).
 
-If a user is an **Organization Admin** or a deployment-level **Admin** or **Editor**, they already have the most permissive user role possible for the code location. In this case, an override is unnecessary and the **Edit user role** button next to the code location will be disabled:
+---
 
-<!-- ![Override for a code location not allowed](/images/dagster-cloud/user-token-management/not-allowed-code-location-override.png) -->
+## Managing users
 
-<Image
-alt="Override for a code location not allowed"
-src="/images/dagster-cloud/user-token-management/not-allowed-code-location-override.png"
-width={2064}
-height={1052}
-/>
-
-To remove an override, click **Edit user role** next to the code location to open the user role selection dialog. Then, click the **Remove override** button:
+<Note>
+  <strong>Organization Admin</strong> permissions are required to manage users
+  in Dagster Cloud.
+</Note>
 
-<!-- ![Removing the override for a code location](/images/dagster-cloud/user-token-management/remove-code-location-override.png) -->
+After a user is created, the **Manage user permissions** window will automatically display. You can also access this window by clicking **Edit** next to a user in the users table.
 
 <Image
-alt="Removing the override for a code location"
-src="/images/dagster-cloud/user-token-management/remove-code-location-override.png"
-width={700}
-height={500}
+alt="Managing user teams and roles for a user in Dagster Cloud"
+src="/images/dagster-cloud/user-token-management/manage-new-user-permissions.png"
+width={1792}
+height={1474}
 />
 
-#### Branch deployment permissions
+### Adding users to teams
 
-Granting access to Branch Deployments gives users access to all Branch Deployments for all the code locations they have access to. Consider the following example:
+<Note>User teams are a Dagster Cloud Enterprise feature.</Note>
```

Comment:
I think we can call them 'teams' throughout, as opposed to 'user teams'

Reply:
Thanks for catching this! It got a little fuzzy at the end.